gh-clone-org

```diff
@@ -85,6 +89,9 @@ then
 elif [ -n "$CLONE_TOPIC" ]
 then
   QUERY="$QUERY%20topic:$CLONE_TOPIC"
+elif [ "$ARCHIVED" != "true" ]
+then
+  QUERY="$QUERY%20archived:false"
```

Comment:
This doesn't feel right to me. As it is now, if I ran the script with `-t my-topic -a` or `-s "my search" -a` the `-a` would be ignored, and if I don't specify any arguments the script would never hit the `else` below this condition.

I also think the argument should also be changed to `--skip-archived` so everything is included by default.